tests/plugins/reporter_backends/test_console.py

```diff
@@ -51,7 +51,7 @@ def test_console_reporter_renderer_envs_list(mocker):
 
     output = console_reporter_renderer.envs_list(["/tmp/envs"])
 
-    assert "envs                   /tmp/envs" in output
```

Comment:
what are we checking here? 🤔

Reply:
Adding the protected marker adds a couple of extra spaces that needed to be adjusted in the existing tests. Arguably we could rewrite this to use a regex though, but 🤷